images/ceph/Makefile

```diff
@@ -18,9 +18,9 @@ include ../image.mk
 # Image Build Options
 
 ifeq ($(GOARCH),amd64)
-CEPH_VERSION ?= v17.2.3-20220804
+CEPH_VERSION ?= v17.2.5-20221017
 else
-CEPH_VERSION ?= v17.2.3-20220805
+CEPH_VERSION ?= v17.2.5-20221017
```

Comment:
@travisn where to ceph release for diff arch?

Reply:
This looks correct. See the arm64 image tags [here](https://quay.io/repository/ceph/ceph-arm64?tag=latest&tab=tags)